frontend/src/components/mine/ContactInfo/PartyRelationships/DefaultContact.js

```diff
@@ -12,71 +14,133 @@ const propTypes = {
   removePartyRelationship: PropTypes.func.isRequired,
+  otherDetails: PropTypes.object,
+  isEditable: PropTypes.bool.isRequired,
 };
 
-export const DefaultContact = (props) => (
-  <div>
-    <div className="inline-flex between">
-      <div>
-        <h4>{props.partyRelationshipTypeLabel}</h4>
-        <p>
-          <Icon type="clock-circle" />
-          &nbsp;&nbsp;
-          {props.partyRelationship.start_date === "None"
-            ? "Unknown"
-            : props.partyRelationship.start_date}{" "}
-          -{" "}
-          {props.partyRelationship.end_date === "None"
-            ? "Present"
-            : props.partyRelationship.end_date}
-        </p>
-        <br />
-      </div>
-      <div>
-        <Button
-          key={`${props.partyRelationship.mine_party_appt_guid}_edit`}
-          ghost
-          type="primary"
-          onClick={() =>
-            props.openEditPartyRelationshipModal(
-              props.partyRelationship,
-              props.onSubmitEditPartyRelationship,
-              props.handleChange,
-              props.mine
-            )
-          }
-        >
-          <img className="padding-small" src={BRAND_PENCIL} />
-        </Button>
-        <Popconfirm
-          key={`${props.partyRelationship.mine_party_appt_guid}_delete`}
-          placement="topLeft"
-          title={`Are you sure you want to delete this ${props.partyRelationshipTypeLabel}?`}
-          onConfirm={(event) =>
-            props.removePartyRelationship(event, props.partyRelationship.mine_party_appt_guid)
-          }
-          okText="Delete"
-          cancelText="Cancel"
-        >
-          <Button ghost type="primary">
-            <Icon type="minus-circle" theme="outlined" />
-          </Button>
-        </Popconfirm>
-      </div>
-    </div>
-    <h5 className="bold">{props.partyRelationship.party.name}</h5>
-    <p>
-      <Icon type="mail" />
-      &nbsp;&nbsp;
-      {props.partyRelationship.party.email}&nbsp;&nbsp;&nbsp;&nbsp;
-      <br />
-      <Icon type="phone" />
-      &nbsp;&nbsp;
-      {props.partyRelationship.party.phone_no}{" "}
-      {props.partyRelationship.party.phone_ext ? `x${props.partyRelationship.party.phone_ext}` : ""}
-    </p>
-  </div>
-);
+export class DefaultContact extends Component {
+  state = {
+    redirectToRelationshipProfile: false,
+    redirectToPartyProfile: false,
+  };
+
+  render() {
+    if (this.state.redirectToRelationshipProfile === true) {
+      return (
+        <Redirect
+          to={router.RELATIONSHIP_PROFILE.dynamicRoute(
+            this.props.mine.guid,
+            this.props.partyRelationship.mine_party_appt_type_code
+          )}
+        />
+      );
+    }
+    if (this.state.redirectToPartyProfile === true) {
+      return (
+        <Redirect
+          to={router.PARTY_PROFILE.dynamicRoute(this.props.partyRelationship.party.party_guid)}
+        />
+      );
+    }
+
+    return (
+      <Card
+        headStyle={{
```

Comment:
This much inline style seems like a code smell

Reply:
It is. The intent is to remove it long term as there is a separate backlog item to implement the style generically for cards and I wanted to spend minimal time on styles as part of this item.